restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/MediaClient/AppRTCAudioManager.java

```diff
@@ -504,8 +505,6 @@ private boolean hasWiredHeadset()
     */
    private void updateAudioDeviceState(boolean hasWiredHeadset)
    {
-      // Update the list of available audio devices.
-      audioDevices.clear();
```

Comment:
This seems dangerous; if we keep plugging and unplugging the headset we'll end up with a lot of audioDevices, with unknown results/behavior